pkg/proxy/ipvs/proxier.go

```diff
@@ -1779,6 +1790,22 @@ func (proxier *Proxier) writeIptablesRules() {
 		"-j", "ACCEPT",
 	)
 
+	// Add rules to the filter/KUBE-IPVS-FILTER chain to prevent access to ports on the host through VIP addresses.
+	// https://github.com/kubernetes/kubernetes/issues/72236
+	proxier.filterRules.Write(
```

Comment:
@danwinship We have historically NOT used ipset in iptables mode, but maybe that's something we should consider?